streams/src/test/java/org/apache/kafka/streams/state/internals/CachingKeyValueStoreTest.java

```diff
@@ -61,14 +62,15 @@ public void setUp() throws Exception {
         store = new CachingKeyValueStore<>(underlyingStore, Serdes.String(), Serdes.String());
         store.setFlushListener(cacheFlushListener);
         cache = new ThreadCache("testCache", maxCacheSizeBytes, new MockStreamsMetrics(new Metrics()));
-        final MockProcessorContext context = new MockProcessorContext(null, null, null, (RecordCollector) null, cache);
+        context = new MockProcessorContext(null, null, null, (RecordCollector) null, cache);
         topic = "topic";
         context.setRecordContext(new ProcessorRecordContext(10, 0, 0, topic));
         store.init(context, null);
     }
 
     @Test
     public void shouldPutGetToFromCache() throws Exception {
+        context.close();
```

Comment:
I don't understand why this is here? Shouldn't we have an `@After` where `context.close()` is called?